app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -1,16 +1,22 @@
 import { render, screen, waitFor } from '@testing-library/react';
 import Entry from './page';
 import { useDataStore } from '@/store/dataStore';
-import { getGameWeek, getCurrentUserEntries } from '@/api/apiFunctions';
+import {
+  getGameWeek,
+  getCurrentUserEntries,
+  getCurrentLeague,
+} from '@/api/apiFunctions';
 
 jest.mock('@/store/dataStore', () => ({
   useDataStore: jest.fn(() => ({ user: { id: '123', leagues: [] } })),
 }));
 
 jest.mock('@/api/apiFunctions', () => ({
-  getGameWeek: jest.fn(() =>
+  getCurrentLeague: jest.fn(() =>
     Promise.resolve({
-      week: 1,
+      leagueName: 'Test League',
+      participants: 12,
+      survivors: 10,
```

Comment:
[getCurrentLeague](https://github.com/LetsGetTechnical/gridiron-survivor/blob/6a4e5bf61a63a251aff009b0a3d1b84a7c30cbac/api/apiFunctions.ts#L158) seems to be returning       leagueId, leagueName, logo, participants, and survivors. Is this correct?

Reply:
> [getCurrentLeague](https://github.com/LetsGetTechnical/gridiron-survivor/blob/6a4e5bf61a63a251aff009b0a3d1b84a7c30cbac/api/apiFunctions.ts#L158) seems to be returning leagueId, leagueName, logo, participants, and survivors. Is this correct?

Yes it is, this is all needed for the information on the page (see screenshots in the PR description).